src/app/core/basic-datatypes/date/date-range-filter/date-range-filter.component.spec.ts

```diff
@@ -101,8 +102,8 @@ describe("DateRangeFilterComponent", () => {
 
   it("should set the correct date filter when changing the date range manually", () => {
     component.filterConfig = new DateFilter("test", "test", []);
-    component.fromDate = new Date("2021-10-28");
-    component.toDate = new Date("2024-02-12");
+    component.fromDate = moment("2021-10-28").toDate();
+    component.toDate = moment("2024-02-12").toDate();
```

Comment:
in practice , these are set using `new Date(selectedOptionDates[0])` only - will this potentially become an issue filtering things?

Reply:
... actual code is fixed as well, using moment